src/Components/RoutineCreator/RoutineCreator.js

```diff
@@ -94,31 +127,60 @@ export class RoutineCreator extends Component {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: '#667D90',
+    backgroundColor: '#2D71A8',
     height: 400,
     width: Dimensions.get('window').width,
     alignItems: 'center'
   },
   item: {
-    color: '#667D90',
-    backgroundColor: '#ACC6D0',
+    color: '#2D71A8',
+    backgroundColor: '#FFFFFF',
     fontFamily: 'raleway-bold'
   },
   input: {
-    color: '#FFFFFF',
-    backgroundColor: '#7C9DB1',
+    color: '#53CFFF',
+    backgroundColor: '#FFFFFF',
     fontFamily: 'raleway-bold',
     width: Dimensions.get('window').width * .90,
-    marginTop: 60,
-    height: 40
+    marginTop: 30,
+    height: 40,
   },
   button: {
-    backgroundColor: '#7C9DB1',
-    width: Dimensions.get('window').width * .90
+    backgroundColor: '#FFFFFF',
+    width: Dimensions.get('window').width * .90,
+    borderRadius: 10,
+    marginTop: 20,
+    height: 40
   },
   text: {
     fontFamily: 'raleway-bold',
-    color: '#FFFFFF'
+    color: '#2D71A8',
+  },
+  introText: {
+    color: palette.lightAccent,
+    fontFamily: 'raleway-bold',
+    fontSize: 20
+  },
+  exercises: {
+    backgroundColor: '#FFFFFF',
+    width: Dimensions.get('window').width * .90,
+  },
+  exerciseItem: {
+    color: '#2D71A8',
+    marginLeft: 10,
+    fontSize: 20
+  },
+  addButton: {
+    backgroundColor: '#FFFFFF',
+    width: Dimensions.get('window').width * .90,
+    borderRadius: 10,
+    height: 40,
+    marginBottom: 10
+  },
+  nameInput: {
+    backgroundColor: '#FFF',
```

Comment:
You have `#FFF` here and `#FFFFFF` on line 174.. let's try to keep uniform here as well and just use `#FFF` wherever possible